scriptworker/worker.py

```diff
@@ -107,9 +107,17 @@ def main():
     with aiohttp.ClientSession(connector=conn) as session:
         context.session = session
         context.credentials = credentials
+        tmp_gpg_home, _ = _get_tmp_gpg_home(context)
+        loop.create_task(
```

Comment:
ayncio noob question: What happens if we don't wait on the task created here? Does it still execute it somehow, but we don't have control on when it happens?